distributed/rpc/batch/requirements.txt

```diff
@@ -0,0 +1,3 @@
+torch
```

Comment:
Do we need to enforce at least v1.6.0 or should it automatically do that?

Reply:
Good point, let me add a check to the code.